includes/class-sensei-quiz.php

```diff
@@ -250,7 +250,7 @@ public static function save_user_answers( $quiz_answers, $files = array(), $less
 			}
 		}
 
-		return $answers_saved;
+		return (bool) $answers_saved;
```

Comment:
Now thinking about it - it's possible but highly unlikely. The method will sometimes return the answers comment meta id (if the meta is created, but not when updated). I just don't see how this could be useful. If a third party needs the meta, it has to do a query anyway, which is done by meta key. 

Maybe a use case is to check if this is the first submission? But this is a very unusual/hidden way to do it. Other, more obvious methods should be used instead.

My idea here was to try and clean up the codebase bit by bit where I see fit and prevent use cases like this one even if there is a very small chance of breaking things.

Let me know if such optimizations are not advised and I will revert that code. 🙂

Reply:
Unfortunately, Sensei has garnered a bit of a reputation over the years for breaking sites on plugin update, and so now we find ourselves in the unenviable position of having to build back that trust, which is hard. 🙁 That has made me a bit sensitive to introducing potentially breaking changes.

Having said that, if you think the benefits of making this change outweigh the associated risk, then I'll happily approve this PR as-is. 🙂

Reply:
Trust is more important than this change. 🙂

Reverted in 63609b42d23f09e9803c27e67842eb05fcfd7ef5.

Reply:
Since this is something that comes up fairly frequently, I Googled around a bit and found this - https://developer.wordpress.org/block-editor/how-to-guides/backward-compatibility/#class-names-and-dom-updates. What do you think about adopting something similar? Unfortunately, I think it would mean adding a new "CSS" GitHub label to flag these types of changes. And adding labels is something that's easily forgotten. 🙁

Reply:
Sounds good to me! I think any process that helps us refactor is nice. Otherwise, we get stuck with code that is hard to work with or doesn't make sense anymore, and that delays the growth and development of the product.